les/coinbase-transaction-v1.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+   "@context": {
+      "@version": 1.1,
+      "cbt": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#",
+      "schema": "http://schema.org/",
+      "CoinbaseTransaction": {
```

Comment:
I wouldn't call it CoinbaseTransaction. Simply CryptoTransaction is fine, as non of it really is coinbase specific

Reply:
changed